### DISCUSSION TITLE
请问ClashMetaForAndroid里面的clash core怎么更新

### DISCUSSION THREAD

**Discussion by Jacky-Bruse** [MetaCubeX/mihomo#970] [Q&A]
查看了一下项目里面好像是一个快捷方式，本人代码方面比较生疏，不知道在什么地方进行更改，请教各位大佬
![IMG_20240110_211859_649~2](https://github.com/MetaCubeX/mihomo/assets/6067707/dcf7d5db-ac43-4b3e-85c3-3b3bd68e0908)

**Comment by makaspacex** [comment]
这是git相关的内容
